packages/cli/src/config/index.ts

```diff
@@ -11,10 +11,13 @@ if (inE2ETests) {
 	process.env = {
 		E2E_TESTS: 'true',
 		N8N_USER_FOLDER: mkdtempSync(join(tmpdir(), 'n8n-e2e-')),
+		EXECUTIONS_PROCESS: 'main',
 		N8N_DIAGNOSTICS_ENABLED: 'false',
 		N8N_PUBLIC_API_DISABLED: 'true',
 		EXTERNAL_FRONTEND_HOOKS_URLS: '',
 		N8N_PERSONALIZATION_ENABLED: 'false',
+		N8N_VERSION_NOTIFICATIONS_ENABLED: 'false',
+		N8N_TEMPLATES_ENABLED: 'false',
```

Comment:
why are you disabling these?

Reply:
as per Adi

Reply:
we need to add tests for these features, not disable them..

Reply:
ok, rolling back the change